packages/plugin/src/lib/paystack.ts

```diff
@@ -40,48 +41,76 @@ export default class Paystack {
     this.apiKey = apiKey;
   }
 
+  // protected async requestPaystackAPI<T>(request: Request): Promise<T> {
```

Comment:
We should take out the previous implementation code completely since it's no longer being used.